action/protocol/execution/evm/evm.go

```diff
@@ -206,6 +222,12 @@ func ExecuteContract(
 	}
 	stateDB.clear()
 	receipt.Logs = stateDB.Logs()
```

Comment:
nit: should we switch order of these 2 lines?
and in clear(), also do statedb.Logs = make([]*action.Log)
maybe another PR